sci-electronics/gnucap-models/Manifest

```diff
@@ -0,0 +1 @@
+DIST gnucap-models-20171003.tar.gz 8048334 BLAKE2B c74a1da922c87dfe78bc3a9249764100221c1ff4ec11a8c1bb2d6f61026dab2fd2cea68c7842973f7239237887eb7825e8d4c97821e6af867ed77240a8206bfb SHA512 40b358554d6bfd579cc388b6c8116bdbf443a11b9b57434f17fc39ddb57f2b5d22fe8847deb4737bd24db004e85503c0bbe639e47df8e293d5c6ed1f08f7f2b8
```

Comment:
I'm sorry but this commit message doesn't conform to our standards. CAT/PN prefix is obligatory.